src/components/infoBox.tsx

```diff
@@ -39,6 +43,14 @@ const InfoBox: React.FC<InfoBoxProps> = ({
       "text-xs": textSize === "small",
       "text-sm": textSize === "medium",
       "text-m": textSize === "large",
+
+      // size
+      "py-6 px-10": size !== "small",
```

Comment:
This declaration could be an issue in the future. For example if you want to add another size like "large". It would be better to check for null